### DISCUSSION TITLE
The Submods folder is deleted when the game starts

### DISCUSSION THREAD

**Discussion by fujori** [Monika-After-Story/MonikaModDev#9955] [QA - tech support + general] [answered]
The Submods folder is deleted all the time when the game starts, I put an exception in the Windows security system, but it didn't help. When I launch DDLC.exe , the Submods folder will be deleted and the game loading process will stop, you can start the game only without the Submods folder in \DDLC\game. The latest version of both the game itself and the mod is installed **(I also use a russifier, but it's not from "incomprehensible" sites, but the one that everyone uses, and Submods work correctly of  everything players)** also the latest possible version of the Submod (in my case, this is YouTube Music), I installed everything according to the instructions. I could not find information on the Internet regarding my problem, please help.
![Screenshot_3](https://user-images.githubusercontent.com/130237222/230765465-3b136ca9-b213-4831-9a7e-cb68c595c345.png)
![Screenshot_4](https://user-images.githubusercontent.com/130237222/230765473-2dc4d8f7-da0f-4439-8edb-c88438e8fcd3.png)

**Comment by Booplicate** [comment, marked as answer]
This doesn't appear to be MAS, we only provide support for the official mod.

**Reply by fujori** [reply]
Thanks for the clarification. I just thought that the problem was in the source code, but it turned out that the developers of the russifier wrote something in the code, they told me so themselves, and explained, like "so that no one would say later, I did something there, and everything broke, help" :D.
In fact, it's sad...
